src/NzbDrone.Core/DecisionEngine/Specifications/Search/TorrentSeedingSpecification.cs

```diff
@@ -25,10 +30,14 @@ public Decision IsSatisfiedBy(RemoteEpisode remoteEpisode, SearchCriteriaBase se
                 return Decision.Accept();
             }
 
-            if (torrentInfo.Seeders != null && torrentInfo.Seeders < 1)
+            var indexer = _indexerFactory.Get(torrentInfo.IndexerId);
+            var minimumSeedersProperty = indexer.Settings.GetType().GetSimpleProperties().SingleOrDefault(s => s.Name == "MinimumSeeders");
+            var minimumSeeders = minimumSeedersProperty == null ? 1 : (int)minimumSeedersProperty.GetValue(indexer.Settings, null);
```

Comment:
I'm not very happy with this, if `MinimumSeeders` was a property on the indexer this could be avoided, but would be extraneous for Usenet indexers (though not really causing harm).